tests/test_state_change_block.py

```diff
@@ -89,15 +89,14 @@ def test_state_change_not_persist(self):
             })
 
         blk.start()
-        #get state, causing it to be saved
+
+        # get state, causing it to be saved
         blk.get_state('A')
+
+        # assert that no values are persisted
         self.assertTrue(len(blk.persisted_values()) == 0)
-        # assert that blk.persisted_values return empty list
-        # set state on block
-        # assert not empty list
         blk.stop()
-        #self.assertTrue('_states' not in blk._persistence._values)
-    
+
     def test_state_change_persist(self):
```

Comment:
PEP-257 - checkout out formatting for multi line docstrings https://www.python.org/dev/peps/pep-0257/#id17